templates/unbound.conf.erb

```diff
@@ -11,6 +11,9 @@ server:
 <% else -%>
   use-syslog: yes
 <% end -%>
+<% if @do_not_query_address -%>
+  do-not-query-address: <%= @do_not_query_address%>
```

Comment:
I think we still need the array loop around this in order to allow multiple values in the array, when its set.